hotelBooking/src/SingleRoom.java

```diff
@@ -0,0 +1,13 @@
+public class SingleRoom extends Hotel implements PricePerRoom{
```

Comment:
SingleRoom extends Room implements Priceable - it's better

Reply:
ok? th u

Reply:
gorod_oop - this is my branch, did it yesterday

Reply:
https://prnt.sc/pwxap2

Reply:
make PR with your new branch and delete unnecessary PRs - pliz have a look on screenshot above.